step-api/step-api-keyword/src/main/java/step/handlers/javahandler/Keyword.java

```diff
@@ -27,6 +27,8 @@
 @Target(ElementType.METHOD)
 public @interface Keyword {
 
+	/** Reserved keyword for the routing element; execution will occur on the controller rather than on agents */
+    String EXECUTE_ON_CONTROLLER = "controller";
```

Comment:
Is the package-private definition intentional? This should be public IMO.
Also, I propose to change the name to ROUTING_EXECUTE_ON_CONTROLLER.

Reply:
Edit: oh, this is an interface, so it's public anyway :)